src/pages/index.js

```diff
@@ -10,18 +10,17 @@ export const client = createClient({
   useCdn: false,
 });
 
-const homeQuery = groq`*[_type == "home"]`;
 const bannerQuery = groq`*[_type == "banner"]`;
-const servicesQuery = groq`*[_type == "home"]`;
+const departmentQuery = groq`*[_type == "department"]`;
 
 export const getStaticProps = async () => {
-  const home = await client.fetch(homeQuery)
   const banner = await client.fetch(bannerQuery)
```

Comment:
why all banners?

Reply:
If banner object is empty, the banner will disappear. In order to add a banner, User will have to add the data in sanity studio.